fscrypt.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: fscrypt
+  version: 0.3.4
```

Comment:
There was a new tagged release this year https://github.com/google/fscrypt/releases/tag/v0.3.5

Reply:
Good catch, looks like they published it as a tag but not a release.